src/components/Carousel/Carousel.js

```diff
@@ -0,0 +1,28 @@
+import React from 'react';
+import '../../App.css';
+import CarouselText from './CarouselText';
+import carousel_img_1 from './images/carousel_img_1.png';
+import carousel_img_2 from './images/carousel_img_2.png';
+import carousel_img_3 from './images/carousel_img_3.png';
+
+class Carousel extends React.Component {
+    render() {
+        return(
+            <div className="carousel">
+                < CarouselText />
+                <div className="carousel__box middle">
```

Comment:
I think you should not hardcore each image, but create carousel dynamically, you are not using React benefits to make reusable components

Reply:
Removed extra images (I haven't done the carousel functionally yet)